test/cmdlineTests/optimizer_no_immutable_assignments/input.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
```

Comment:
The test requires `--optimize` flag. What is the best place? I do not really like extending `cmdlineTests`.

Reply:
Even though it is not a syntax test, it might still work. We do something similar for the code size. You can enable the yul optimizer, that might be enough.